src/components/home.js

```diff
@@ -17,8 +17,8 @@ import { SyncLoader } from "react-spinners";
 
 // These are the API URL's
 // const API_URL = 'https://video-cut-tool-back-end.herokuapp.com'
-// const API_URL = "http://localhost:4000"
-const API_URL = 'https://tools.wmflabs.org/video-cut-tool-back-end'
+const API_URL = "http://localhost:4000"
```

Comment:
Ahh it's my bad, will have to improve this using some env variables, But for now can you please comment this line?